modules/ROOT/pages/apikit-odatav2-extension-reference.adoc

```diff
@@ -0,0 +1,41 @@
+= APIkit for OData v2 Module Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+This reference contains dependency, namespace, and schema information.
+
+== Dependency Information
+
+The APIkit for OData v2 extension must be declared with the APIkit for REST module dependency:
```

Comment:
Suggestion (more direct) 

Declare the APIkit for OData v2 extension...

Reply:
OK! Thanks!